cookiecutter.json

```diff
@@ -4,15 +4,16 @@
   "project_owner_github_username": "sbillinge",
   "contributors": "Billinge Group members and community contributors",
   "license_holders": "The Trustees of Columbia University in the City of New York",
-  "github_org": "diffpy",
   "project_name": "diffpy.my-project",
+  "github_org": "diffpy",
+  "repo_name": "diffpy.my-project",
   "package_dist_name": "{{ cookiecutter.project_name|replace(' ', '-')|lower }}",
   "package_dir_name": "{{ cookiecutter.project_name|replace(' ', '_')|replace('-', '_')|lower }}",
-  "keywords": "",
   "project_short_description": "Python package for doing science.",
   "keywords": "diffraction, PDF, X-ray, neutron",
   "minimum_supported_python_version": "3.11",
   "maximum_supported_python_version": "3.13",
-  "have_c_code": ["n", "y"],
+  "project_needs_c_code_compiled": ["No", "Yes"],
+  "project_needs_headless_GUI_testing": ["No", "Yes"],
```

Comment:
`project_has_gui_tests`?

Reply:
is the capitalization of Yes ano relevant?

Reply:
done

> is the capitalization of Yes ano relevant?

Nope, user choose (1) or (2), and default is (1) which is "No"

![Uploading Screenshot 2025-01-02 at 2.38.56 PM.png…]()